Module 5/Problem_18/Q18.json

```diff
@@ -0,0 +1,61 @@
+Input JSON
+{
+  "employees": [
+    {
+      "name": "John",
+      "age": 30,
+      "salary": 5000
+    },
+    {
+      "name": "Jane",
+      "age": 35,
+      "salary": 6000
+    },
+    {
+      "name": "Bob",
+      "age": 40,
+      "salary": 7000
+    }
+  ]
+}
+
+Jolt Spec
+[
+  {
+    "operation": "modify-overwrite-beta",
+    "spec": {
+      "employees": {
+        "*": {
+          "temp": "=divideAndRound(0,@(1,salary),10)",
+          "salary": "=doubleSum(@(1,salary),@(1,temp))"
+        }
+      }
+    }
+  },
+  {
+    "operation": "remove",
+    "spec": {
+      "temp": ""
+    }
+  }
+] 
+
+Output JSON
+{
+  "employees" : [ {
+    "name" : "John",
+    "age" : 30,
+    "salary" : 5500.0,
```

Comment:
salary should be in Integer and why there is temp in output ?

Reply:
Made the required changes.